com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/v1/SpecialResourceTypesRequiredParameters.java

```diff
@@ -0,0 +1,34 @@
+package com.sap.cloud.lm.sl.cf.core.cf.v1;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import com.sap.cloud.lm.sl.cf.core.model.SupportedParameters;
+
+public enum SpecialResourceTypesRequiredParameters {
```

Comment:
I would actually merge this into the ResourceType enum. Then, the getRequiredParameters() method would be much more simple. Actually, it will be just a getter. :)